lnbits/app.py

```diff
@@ -526,6 +526,21 @@ def configure_logger() -> None:
     log_level: str = "DEBUG" if settings.debug else "INFO"
     formatter = Formatter()
     logger.add(sys.stdout, level=log_level, format=formatter.format)
+
+    if settings.log_file:
+        logger.add(
+            Path(settings.lnbits_data_folder, "lnbits.log"),
```

Comment:
```suggestion
Path(settings.lnbits_data_folder, "logs", "lnbits.log"),
```
- do not store all log files on the top level